manifests/service.pp

```diff
@@ -5,30 +5,30 @@
 # == Authors
 #
 # * Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
-#
+
 class sssd::service {
```

Comment:
This needs to have `assert_private()`. It did not previously and could be use standalone.